src/services/pocket-relayer.ts

```diff
@@ -773,7 +781,11 @@ export class PocketRelayer {
     blockchainID: string,
     logLimitBlocks: number
   ): Promise<string | Error> {
-    if (parsedRawData.method === 'eth_getLogs') {
+    if (WS_ONLY_METHODS.includes(parsedRawData.method)) {
+      return new HttpErrors.BadRequest(
+        `We cannot serve ${parsedRawData.method} method over HTTPS. At the moment, we do not support WebSockets.`
```

Comment:
I would only change the message to `${parsedRawData.method} method cannot be served over HTTPS. WebSockets are not supported  at moment.`